easybuild/easyconfigs/f/FFLAS-FFPACK/FFLAS-FFPACK-2.2.0-foss-2016a.eb

```diff
@@ -0,0 +1,46 @@
+##
+# This file is an EasyBuild recipe; see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright (c) 2016 Riccardo Murri <riccardo.murri@gmail.com>
+# Authors::   Riccardo Murri <riccardo.murri@gmail.com>
+# License::   GPL
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'FFLAS-FFPACK'
+version = '2.2.0'
+
+homepage = 'https://linbox-team.github.io/fflas-ffpack/'
+description = "Finite Field Linear Algebra Subroutines / Package"
+
+toolchain = {'version': '2016a', 'name': 'foss'}
+
+sources = ['v2.2.0.zip']
+source_urls = ['https://github.com/linbox-team/%(namelower)s/archive']
+
+builddependencies = [
+    ('Autoconf', '2.69', '', True),
+    ('Automake', '1.15', '', True),
+    ('libtool', '2.4.6', '', True),
+]
+dependencies = [
+    ('GMP', '6.1.0', '', ('GCC', '4.9.3-2.25')),
+    ('Givaro', '4.0.1'),
+]
+
+preconfigopts = "env NOCONFIGURE=1 ./autogen.sh && "
```

Comment:
> > +preconfigopts = "env NOCONFIGURE=1 ./autogen.sh && "
>
> I understand that the `NOCONFIGURE=1` is required, but I don't think the `env` command is required at all?

It doesn't do any harm just ensures the `NOCONFIGURE` is set to `1` in
`autogen.sh` environment.  Why not just leave it there?  It makes the
command-line more robust (e.g., if executed with `tcsh` it would still
work as intended).